### PR TITLE
Add rateLimited error

### PR DESCRIPTION
**Pull request by jsha**
And clarify that error list is not exhaustive.

### CODE REVIEW

File: draft-ietf-acme-acme.md [hunk 1]
```diff
@@ -517,6 +517,11 @@ to errors, this document defines the following standard tokens for use in the
 | tls             | The server experienced a TLS error during DV             |
 | unauthorized    | The client lacks sufficient authorization                |
 | unknownHost     | The server could not resolve a domain name               |
+| rateLimited     | The request exceeds a rate limit                         |
+
+This list is not exhaustive. The server MAY return errors whose "type"
```
Comment (kuba) [inline draft-ietf-acme-acme.md]: I think that server should not return anything in `urn:acme:` subspace unless it's registered with the spec

Comment (bifurcation) [inline draft-ietf-acme-acme.md]: This can go in a separate PR.

**Comment by bifurcation** [comment]
Closing in favor of #42
